### ISSUE TITLE
Simple custom resolution configuration feature request (Rice plugin at least)

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#54]
Originally reported on Google Code with ID 54

```
Describe your system:
 - Ubuntu Hardy 8.04, x86_64, Mupen64plus 1.3, Rice/JttL/Blight

Describe the problem:
 - I run a 1280x720 display, and would like the option of (1) being able to
choose a custom resolution, and (2) keeping the N64 4:3 aspect ratio by
pillarboxing the output for people like me with non-4:3 displays.

Please provide any additional information below.
 - This would make my N64 gaming life much better!

```

Reported by `larson.eric.d` on 2008-04-14 23:28:45

**Comment by richard42** [comment]
```
Apparently I marked this a "defect"---didn't mean to do that. As much as
output-stretching bothers me, it's not quite at the "defect" level.
```

Reported by `larson.eric.d` on 2008-04-14 23:30:19

**Comment by richard42** [comment]
```
This is a duplicate of #29.  Thanks for your input.
```

Reported by `richard42g` on 2008-04-15 02:07:04
- **Labels added**: Type-Enhancement
- **Labels removed**: Type-Defect

**Comment by richard42** [comment]
```
My bad. I looked through the list briefly and searched for "resolution", which didn't
match the other post which used the plural term (stupid google).
```

Reported by `larson.eric.d` on 2008-04-15 02:33:49

**Comment by richard42** [comment]
Reported by `richard42g` on 2010-01-08 05:18:27
- **Status changed**: `Duplicate`
- **Merged into**: #29
